readme/doc/index.mdx

```diff
@@ -2,4 +2,169 @@ import { mdx } from '@mdx-js/react';
 import { MdxLayout } from '~/layout';
 export default MdxLayout;
 
-### TO DO
+[![backlight.dev](https://img.shields.io/badge/Backlight.dev-Design%20System%20Editor-%23f8c307)](https://backlight.dev)
+[![webcomponents.dev](https://img.shields.io/badge/WebComponents.dev-Web%20Components%20Studio-%23459BCF)](https://webcomponents.dev)
+[![components.studio](https://img.shields.io/badge/Components.studio-Components%20Studio-%23319795)](https://components.studio)
+
+[![Github Repo](https://img.shields.io/github/last-commit/divriots/starter-react-sui)](https://github.com/divriots/starter-react-sui)
+
+# System UI Tech Sample _by_ ‹div›RIOTS
+
+This is a _Technology Sample_ you can rely on to build your own Design System, based on React and a [System UI theme](https://system-ui.com/theme).
+
```

Comment:
This readme should be opened in Github so we need:

- badge here for "Open in | Backlight.dev"

Reply:
Erratum: We need a copy of this in `README.md` with the "Open in | Backlight.dev"